lib/active_resource/connection_ext.rb

```diff
@@ -5,6 +5,15 @@ class Connection
 
     attr_reader :response
 
+    alias_method :raw_initialize, :initialize
```

Comment:
thought that was best practice for modifying a method without overriding it; I don't think `super` would work here.

I could copy the actual `initialize` body, but I think it's better to alias the old one and call the alias.